pkg/virt-operator/util/readycheck.go

```diff
@@ -88,12 +88,9 @@ func DeploymentIsReady(kv *v1.KubeVirt, deployment *appsv1.Deployment, stores St
 		return false
 	}
 
-	var specReplicas int32 = 1
-	if deployment.Spec.Replicas != nil {
-		specReplicas = *deployment.Spec.Replicas
-	}
-	if specReplicas != deployment.Status.Replicas ||
-		deployment.Status.Replicas != deployment.Status.ReadyReplicas {
+	var requiredSpecReplicas int32 = 1
+	if requiredSpecReplicas > deployment.Status.Replicas ||
```

Comment:
since requiredSpecReplicas is always 1 now, it seems like it would make more sense to remove the varable all together and just have

```if deployment.Status.Replicas == 0 || deployment.Status.ReadyReplicas == 0 ```